spec/variable_fields/empty_subfield_spec.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'marc'
+require 'marc_cleanup'
+require 'byebug'
```

Comment:
Do you want to leave the 'byebug' require? If we want it generally available in the specs, maybe we should add it to the spec_helper?